src/Microsoft.AspNet.Mvc.ViewFeatures/Controller.cs

```diff
@@ -1181,6 +1183,128 @@ public virtual CreatedAtRouteResult CreatedAtRoute(string routeName, object rout
         }
 
         /// <summary>
+        /// Creates a <see cref="ChallengeResult"/>.
+        /// </summary>
+        /// <returns>The created <see cref="ChallengeResult"/> for the response.</returns>
+        [NonAction]
+        public virtual ChallengeResult Challenge()
+            => new ChallengeResult();
+
+        /// <summary>
+        /// Creates a <see cref="ChallengeResult"/> with the specified authentication scheme.
+        /// </summary>
+        /// <param name="authenticationScheme">The authentication scheme to challenge.</param>
+        /// <returns>The created <see cref="ChallengeResult"/> for the response.</returns>
+        [NonAction]
+        public virtual ChallengeResult Challenge(string authenticationScheme)
+            => new ChallengeResult(authenticationScheme);
+
+        /// <summary>
+        /// Creates a <see cref="ChallengeResult"/> with the specified authentication schemes.
+        /// </summary>
+        /// <param name="authenticationScheme">The authentication schemes to challenge.</param>
+        /// <returns>The created <see cref="ChallengeResult"/> for the response.</returns>
+        [NonAction]
+        public virtual ChallengeResult Challenge(IList<string> authenticationSchemes)
```

Comment:
@pranavkm and I chatted and I said "eh, why not" have them all... or are some of them truly more obscure?